docs/specs/iota_did_method_spec.md

```diff
@@ -0,0 +1,324 @@
+
+#### Key Collections
+
+Instead of storing individual public keys in a DID Document, IOTA Identity introduces the `MerkleKeyCollection2021` verification method. It supports a REQUIRED `publicKeyBase58` field that MUST contain the top hash of a Merkle Tree. In this Merkle Tree all the individual leaves are a public keys using the Signature Algorithm of choice and Digest Algorithm of choice for the Merkle Tree process. This process allows the creation of millions of public keys, within a single verification Method and without bloating the DID Document. For specific info such as the maximum depth of the Merkle Tree, supported signature algorithms and digest algorithms can be found in the specification document. 
+
+#### Verifiable Credential Proofs
+
+In addition to the normal signature, a `MerkleKeySignature2021` requires additional proof data to validate the signature's origin and validaty. The public key itself needs to be revealed inside the `signatureValue` field, but also the individual hashes inside the Merkle Tree and their relative location in the path to create a valid `Proof of Inclusion`. 
+
+#### Revocation List
+
+In order to revoke a public key, and therefore any Verifiable Credential or other statements, the DID Document must be updated with a revocation list. The REQUIRED `revocation` field is introduced inside the `MerkleKeyCollection2021` verification method, which lists the indices from the leaves of the Merkle Tree that are revoked. These indices are further compressed via [Roaring Bitmaps](https://roaringbitmap.org/).
+
+### Standardized Services
+
+The IOTA Identity framework also standardized certain `services` that are embedded in the DID Document. It is RECOMMENDED to implement these when implementing the `did:iota` method. 
+
+Currently standardized `services`:
+* Nothing yet. 
+
+## Security Considerations
+
+The `did:iota` method is implemented on the IOTA Tangle, a public permissionless and feeless Distributed Ledger Technology (DLT), making it resistant against almost all censorship attack vectors. Up until the `Coordicide` update for the IOTA network, a reliability on the coordinator exists for resolving ordering conflicts. This has a minor censorship possibility, that, in the worst case, can prevent ordering conflicts from being resolving in a DID. However, these can only be published by the possessor of the private key and therefore does not provide a usable attack vector. Lastly, a node may decide to censor DID messages locally, however a user can easily use another node. 
```

Comment:
Maybe link IOTA Tangle to iota.org?
line 3: from being resolving in a -> from resolving to a
possessor -> owner
does not provide a -> do not constitute a